src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  // Dividir a string em declarações separadas por ';'
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    // Dividir cada declaração em chave e valor
+    const [property, value] = declaration.split(':').map((part) => part.trim());
```

Comment:
The current implementation does not handle cases where there are multiple colons within a property value, which is acceptable as per the task requirements. However, ensure that this is the intended behavior.